src/api/mappers/tagMapper.ts

```diff
@@ -1,12 +1,13 @@
 import { Tag } from '../../models/tag';
+import { VisualNovelTag } from '../../models/visualNovel';
 import { TagClassification } from '../../utils/types/tagClassification';
 import { TagDto } from '../dtos/tagDto';
 
 /**
  * Maps dto into model.
  * @param dto Tag dto.
  */
-export const tagFromDto = (dto: TagDto): Tag => ({
+export const tagFromDto = (dto: TagDto, vnTag: VisualNovelTag): Tag => ({
```

Comment:
Missing param in docs

Reply:
Btw, pls update docs overall and improve namings of params, I don't unterstand why you need `vnTag` here)

Reply:
Well, it seems like an antipattern to pass some other things into `fromDto` or `toDto` method params, I think mapper should be able to map things on its own.

Reply:
I think it would be better to create separate model for tags meta info, and then somehow combine it into one big model

Reply:
Oh no...😿 Abstraction leaked 💦